internal/webhook/envtestutil/webhook_registration.go

```diff
@@ -40,5 +42,15 @@ func RegisterHardwareProfileAndKueueWebhooks(mgr manager.Manager) error {
 		return err
 	}
 
+	// Register Connection webhook for InferenceService
+	connectionWebhook := &inferenceservicewebhook.ConnectionWebhook{
+		Client:  mgr.GetAPIReader(),
+		Decoder: admission.NewDecoder(mgr.GetScheme()),
+		Name:    "connection-isvc",
+	}
+	if err := connectionWebhook.SetupWithManager(mgr); err != nil {
+		return err
+	}
+
```

Comment:
Will there ever be case where we just want the isvc webhook for testing? If not, should we change the name of the function to reflect the new webhook(s)?